src/bigml/api/core.clj

```diff
@@ -41,6 +41,13 @@
   `(binding [*connection* (merge *connection* {:dev-mode true})]
      ~@body))
 
+(defmacro with-mode
```

Comment:
Looking at the tests, right now you only use `with-mode` enabled.  So you could just use `with-dev-mode` as it already is.